lnplus/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: lnplus_web_1
+      APP_PORT: "3777"
+      API_URL: "https://lightningnetwork.plus/api/2/"
```

Comment:
This env. var. is being set for the App Proxy service which doesn't accept `API_URL`. I assume this is supposed to be for your `web` service?

I just tested a few things on `arm64` and `amd64`, and I didn't see any issue, so I assume your `web` service is using a default value for `API_URL`? And as this isn't being used by the App Proxy, this is safe to drop this line. Or would you like to move this to the `web` service?

Reply:
@nevets963 Thanks for educating me on this. I updated both the app and the umbrel-apps repo.